vllm/lora/punica.py

```diff
@@ -450,41 +450,35 @@ def expand_slice_decode(
         bgmv_expand_slice(x, w_t_all, y, self.token_lora_indices, y_offset,
                           y_slice_size, add_input)
 
-    def apply_bias(
-        self,
-        indices: torch.Tensor,
-        output: torch.Tensor,
-        bias_stacked: torch.Tensor,
-    ):
-        """Applies bias to output
-
-        Input shapes:
-            bias_stacked:    (num_loras, output_dim)
-            indices:         (batch_size)
-            output:          (batch_size, output_dim)
+    def apply_expand(self,
```

Comment:
```suggestion
    def _apply_expand(self,
```

TBH, I sometimes feel confused about the methods like `apply_bias` etc, because it is just an internal method used in `PunicaWrapper`, but it quite looks like a method that we will call with `apply` in `layers.py`.

Perhaps renaming it to `_apply` would be clearer?

Reply:
Make sense. 
BTW, I have a  prototype that modifies the punica wrapper, you can take a look at it, see: https://github.com/jeejeelee/vllm/tree/refactor-punica/vllm/lora